packages/core/src/progress-component.ts

```diff
@@ -0,0 +1,375 @@
+/* NProgress, (c) 2013, 2014 Rico Sta. Cruz - http://ricostacruz.com/nprogress
+ * @license MIT */
+
+import { ProgressSettings } from './types'
+
+const baseComponentSelector = 'nprogress'
+
+const settings: ProgressSettings = {
+  minimum: 0.08,
+  easing: 'linear',
+  positionUsing: 'translate3d',
+  speed: 200,
+  trickle: true,
+  trickleSpeed: 200,
+  showSpinner: true,
```

Comment:
Kind of wish we could just remove the spinner entirely. Wish I knew how many people were actually using it.

Reply:
Apparently some people like it: #96